pkgs/development/python-modules/brian2/default.nix

```diff
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, cython
+, jinja2
+, numpy
+, pyparsing
+, setuptools
+, sympy
+}:
+
+buildPythonPackage rec {
+  pname = "Brian2";
+  version = "2.5.1";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-x1EcS7PFCsjPYsq3Lt87SJRW4J5DE/OfdFs3NuyHiLw=";
+  };
+
+  propagatedBuildInputs = [
+    cython
+    jinja2
+    numpy
+    pyparsing
+    setuptools
+    sympy
+  ];
+
+  # fails with TypeError
```

Comment:
What kind of type error? Can you paist the errror?

Reply:
Fixed.